users/documentation/case-studies.md

```diff
@@ -31,6 +31,28 @@ for code, Creative Commons for text)</span>; authors retain all copyright.
 <hr style="margin:0.25em 0 0.25em 0;"/>
 <hr style="margin:0 0 2em 0;"/>
 
+## Estimating Dynamic Cross-National Opinion Using Existing Survey Data
+<small> This case study describes a method for estimating dynamic, cross-national, latent public opinion using existing survey data. It outlines the model and its coding in Stan, explains how to set the model up in R and run it using cmdstanr, then walks through an example where the model is used to estimate public support for democracy in 144 countries from 1988 to 2020.
+
+[View](case-studies/estimating-dynamic-latent-opinion.html) <span
+class="note">(HTML)</span>
+
+Authors
+: Christopher Claassen
+
+Keywords
+: latent variable models, public opinion, longitudinal model
+
+Source Repository
+: [example-models/knitr/dynamic-opinion](https://github.com/joonho112/example-models/tree/master/knitr/dynamic_opinion/estimating-dynamic-latent-opinion.html)
```

Comment:
I'm still getting an error even with this new link. It doesn't look like there's a dynamic-opinion folder inside `joonho112/example-models/tree/master/knitr/`.

Reply:
I've changed the url to my repo - maybe the path of least resistance?

Reply:
Sorry for the delay on this. The current URL I see is 

https://github.com/joonho112/example-models/tree/master/knitr/dynamic_opinion/estimating-dynamic-latent-opinion.html

but this still gives me a 404 - page not found error. Is the URL right? Or is this a private repository maybe?